src/macaron/slsa_analyzer/git_url.py

```diff
@@ -166,36 +166,13 @@ def check_out_repo_target(
     bool
         True if succeed else False.
     """
-    if not offline_mode:
```

Comment:
Could you please also update the doc strings for `check_out_repo_target` and `clone_remote_repo` to reflect the changes?